test/jdk/com/sun/jdi/InvokeHangTest.java

```diff
@@ -59,6 +59,12 @@ public static void main(String[] args) {
 
         t1.start();
         t2.start();
+        try {
+            t1.join();
+            t2.join();
+        } catch (InterruptedException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
Please consider adding a comment that explains that the `join()` calls
are only necessary when `t1` and `t2` are virtual threads.